### PR TITLE
Rename TextFormat to HttpTextFormat, and related.

### PR DESCRIPTION
**Pull request by carlosalberto**
Addresses #127

### CODE REVIEW

**Comment by carlosalberto** [comment]
I'm wondering if more details about complying with HTTP/1.1 are required ;)

File: api/src/main/java/openconsensus/context/propagation/HttpTextFormat.java [hunk 1]
```diff
@@ -21,7 +21,8 @@
 
 /**
  * Injects and extracts a value as text into carriers that travel in-band across process boundaries.
- * Values are often encoded as RPC/HTTP request headers.
+ * Encoding is expected to conform to the HTTP/1.1 Header Field semantics. Values are often encoded
+ * as RPC/HTTP request headers.
  *
  * <p>When using http, the carrier of propagated data on both the client (injector) and server
```
Comment (SergeyKanzhelev) [inline api/src/main/java/openconsensus/context/propagation/HttpTextFormat.java]: perhaps beggiing of this sentence is not needed here

File: api/src/main/java/openconsensus/context/propagation/HttpTextFormat.java [hunk 2]
```diff
@@ -21,7 +21,8 @@
 
 /**
  * Injects and extracts a value as text into carriers that travel in-band across process boundaries.
- * Values are often encoded as RPC/HTTP request headers.
+ * Encoding is expected to conform to the HTTP/1.1 Header Field semantics. Values are often encoded
```
Comment (SergeyKanzhelev) [inline api/src/main/java/openconsensus/context/propagation/HttpTextFormat.java]: is there difference with 1.0 and 2.0? Perhaps this comment narrowing the use of this class to much

**Comment by carlosalberto** [comment]
hey @SergeyKanzhelev so all the documentation inline is updated. A new class for text format as well? I don't think so, based on the comments from #127

File: api/src/main/java/openconsensus/context/propagation/HttpTextFormat.java (hunk 2 above)
Comment (carlosalberto) [reply api/src/main/java/openconsensus/context/propagation/HttpTextFormat.java]: 2.0 adds binary protocol, and that's pretty much it, IIRC

File: api/src/main/java/openconsensus/context/propagation/HttpTextFormat.java (hunk 1 above)
Comment (carlosalberto) [reply api/src/main/java/openconsensus/context/propagation/HttpTextFormat.java]: "Encoding is expected to conform to the HTTP/1.1 Header Field semantics." ? 

We need to make sure that users know this, and why we don't have separated text-map/http-headers formats.

Comment (SergeyKanzhelev) [reply api/src/main/java/openconsensus/context/propagation/HttpTextFormat.java]: It was a comment about this:

> When using http,

File: api/src/main/java/openconsensus/context/propagation/HttpTextFormat.java (hunk 2 above)
Comment (SergeyKanzhelev) [reply api/src/main/java/openconsensus/context/propagation/HttpTextFormat.java]: So should we specify that this works only for HTTP/1.1? Can comment be more generic and say simply HTTP?

File: api/src/main/java/openconsensus/context/propagation/HttpTextFormat.java (hunk 1 above)
Comment (carlosalberto) [reply api/src/main/java/openconsensus/context/propagation/HttpTextFormat.java]: Aha, gotcha, my bad. Makes sense ;)

File: api/src/main/java/openconsensus/context/propagation/HttpTextFormat.java (hunk 2 above)
Comment (carlosalberto) [reply api/src/main/java/openconsensus/context/propagation/HttpTextFormat.java]: I see your point. Yeah, totally, lets go the generic way.

**Comment by carlosalberto** [comment]
Done ;)

**Comment by carlosalberto** [comment]
Hey @bogdandrutu this is now rebased. Shall we merge?

**Pull request merged**